### ISSUE TITLE
404 when going to githubschool.github.io

### ISSUE THREAD

**Issue by Darkphibre** [githubschool/github-games#9]
Just a heads up, I was going to point this resource to a machine learning group call with 200 coworkers... but it's 404!

**Comment by Yubraj977** [comment]
Yep exactly, i come up here because of some college teaching examples but its 404
